lib/compat/wordpress-6.7/rest-api.php

```diff
@@ -114,3 +128,26 @@ function gutenberg_override_default_rest_server() {
 	return 'Gutenberg_REST_Server';
 }
 add_filter( 'wp_rest_server_class', 'gutenberg_override_default_rest_server', 1 );
+
+
+/**
+ * Filters the arguments for registering a wp_global_styles post type.
+ * Note when syncing to Core: the capabilities should be updates for `wp_global_styles` in the wp-includes/post.php.
+ *
+ * @since 6.7.0
+ *
+ * @param array  $args      Array of arguments for registering a post type.
+ *                          See the register_post_type() function for accepted arguments.
+ * @param string $post_type Post type key.
+ *
+ * @return array|string Array or string of arguments for registering a post type.
```

Comment:
```suggestion
 * @return array Array of arguments for registering a post type.
```

This filter shouldn't return a `string`.

Reply:
Will do. I did a straight copy paste from the [consuming method](https://github.com/WordPress/wordpress-develop/blob/47122105f20624e5a6b8ed151d732157f2f2605c/src/wp-includes/class-wp-post-type.php#L477)